docs/source/settings.rst

```diff
@@ -119,10 +119,12 @@ you might want to choose one of the other worker classes.
 A string referring to one of the following bundled classes:
 
 * ``sync``
-* ``eventlet`` - Requires eventlet >= 0.9.7
-* ``gevent``   - Requires gevent >= 0.13
-* ``tornado``  - Requires tornado >= 0.2
-* ``gthread``  - Python 2 requires the futures package to be installed
+* ``eventlet`` - Requires eventlet >= 0.9.7 (or install using
```

Comment:
Could you please make these changes in `gunicorn/config.py` and then run `make html` in `docs` directory? (and please  don't include unrelated changes to `docs/source/settings.rst` after you ran `make html`)

Reply:
indeed that should be done in settings.py in any case

Reply:
> please don't include unrelated changes to `docs/source/settings.rst` after you ran `make html`

Looking at the generated `settings.rst`, I'm not sure what you mean. Do you mean I should not commit it at all?

Reply:
After you ran `make html`, if you do `git diff` on `settings.rst` you will see changes like:

```diff
diff --git a/docs/source/settings.rst b/docs/source/settings.rst
index 4a8d027..738e18a 100644
--- a/docs/source/settings.rst
+++ b/docs/source/settings.rst
@@ -450,7 +450,7 @@ chdir
 ~~~~~
 
 * ``--chdir``
-* ``/Users/benoitc/work/gunicorn/py27/gunicorn/docs``
+* ``/home/berker/projects/gunicorn/docs``
 
 Chdir to specified directory before apps loading.
 
@@ -521,7 +521,7 @@ user
 ~~~~
 
 * ``-u USER, --user USER``
-* ``501``
+* ``1000``
 
 Switch worker processes to run as this user.
 
@@ -535,7 +535,7 @@ group
 ~~~~~
 
 * ``-g GROUP, --group GROUP``
-* ``20``
+* ``1000``
 
 Switch worker process to run as this group.
 
@@ -594,7 +594,7 @@ temporary directory.
 secure_scheme_headers
 ~~~~~~~~~~~~~~~~~~~~~
 
-* ``{'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}``
+* ``{'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on', 'X-FORWARDED-PROTOCOL': 'ssl'}``
 
 A dictionary containing headers and values that the front-end proxy
 uses to indicate HTTPS requests. These tell Gunicorn to set
@@ -763,13 +763,30 @@ The log config file to use.
 Gunicorn uses the standard Python logging module's Configuration
 file format.

```

I was talking about these unrelated changes in my previous comment. They are created by `docs/gunicorn_ext.py` but we usually discard them when we generate `settings.rst` for documentation changes.

Reply:
Ok, thanks for the clarification. Should be better now.